lib/_http_server.js

```diff
@@ -454,8 +454,11 @@ function socketOnError(e) {
   this.removeListener('error', socketOnError);
   this.on('error', () => {});
 
-  if (!this.server.emit('clientError', e, this))
+  if (!this.server.emit('clientError', e, this)) {
+    if (this.writable)
+      this.write('HTTP/1.1 400 ' + STATUS_CODES[400] + CRLF + CRLF, 'ascii');
```

Comment:
It would be nice to change this to a static value instead of using string concatenation as @mscdex pointed out.